vermouth/tests/test_annotate_mut_mod.py

```diff
@@ -289,7 +291,8 @@ def test_nter_cter_modifications(node_data, edge_data, expected):
     mol = Molecule(force_field=ForceField(FF_UNIVERSAL_TEST))
     mol.add_nodes_from(enumerate(node_data))
     mol.add_edges_from(edge_data)
-    modification = [({'resname': 'cter'}, 'C-ter'), ({'resname': 'nter'}, 'N-ter')]
+    modification = [({'resname': 'cter', 'resid': 3}, 'C-ter'), 
+                    ({'resname': 'nter', 'resid': 1}, 'N-ter')]
```

Comment:
Right, as in the comment before, not all resspecs need a resid or resname. These for example don't need a resid; the 'nter' and 'cter' already have enough information to not need the resid or sane resname.

Reply:
weirdly, adding those resids seemed to be what fixed the tests before, but I just tried again with them removed and it looks like it works without? I think we thought it was overwriting the whole dictionary now, and that's why they were needed. Not sure what's happened there.